### DISCUSSION TITLE
higress配置了gitlab域名后，如何使gitlab中配置的ssh key能正常使用

### DISCUSSION THREAD

**Discussion by blpwa** [higress-group/higress#929] [Q&A]
1.我使用docker安装了gitlab
2.higress服务来源中增加了以固定地址形式的gitlab服务来源
3.higress中增加了对应域名
4.higress中增加了gitlab路由配置

完成这些配置后gitlab能够通过域名形式正常访问，但是在gitlab中配置ssh key后，clone代码还是提示要输入密码，也就是配置的ssh key没达到效果，请问下要使ssh key配置后能正常使用，还需要做哪些配置的修改

**Comment by CH3CHO** [comment]
这不是一个协议呀。你是怎么 clone 的？

**Reply by blpwa** [reply]
就是没clone成功，还是要使用用户名密码的形式

**Reply by CH3CHO** [reply]
你用的是HTTP的方式Clone的还是用的SSH？

**Reply by blpwa** [reply]
用的SSH方式CLone的

**Reply by CH3CHO** [reply]
那SSH端口也走了Higress代理吗？

**Reply by blpwa** [reply]
在Higress文档上没找到关于SSH端口代理配置方式，有相关文档或者配置方式吗
